src/main/java/core/basesyntax/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Map;
```

Comment:
Please organize your classes into packages by their purpose/common logic. The `BalanceOperation` class should be in the `strategy` package, as it is part of the Strategy pattern implementation.